web/src/components/ShelterInformationSection.tsx

```diff
@@ -89,25 +90,37 @@ const ShelterInformationSection = ({
   information,
   children,
   extended,
-  elevated = false,
-  separationLine = false
+  elevated = false
 }: Props): ReactElement => {
   const { viewportSmall } = useWindowDimensions()
   return (
-    <Container separationLine={separationLine} extended={extended} elevated={elevated}>
+    <Container extended={extended} elevated={elevated}>
       <Row>
         <Title>{title}</Title>
         {titleHint && <TitleHint>{titleHint}</TitleHint>}
         {label && <Label>{label}</Label>}
       </Row>
       <RowDetail viewportSmall={viewportSmall}>
-        {information.map(({ text, icon, rightText, link }) => (
-          <Detail key={text} extended={extended} as={link ? CleanLink : 'div'} to={link}>
-            {icon && <img alt='' src={icon} />}
-            <DetailText hasText={!!rightText}>{text}</DetailText>
-            {rightText && <RightTextContainer>{rightText}</RightTextContainer>}
-          </Detail>
-        ))}
+        {information.map(({ text, icon, rightText, link, tooltip }) => {
```

Comment:
its really getting hard to read here :/ But okay for the moment. Maybe extract it to separate component in the future